src/flip.sol

```diff
@@ -57,19 +61,20 @@ contract Flipper is LibNote {
         uint48  end;  // auction expiry time      [unix epoch time]
         address usr;
         address gal;
-        uint256 tab;  // total dai wanted    [rad]
+        uint256 tab;  // total dai wanted         [rad]
     }
 
-    mapping (uint => Bid) public bids;
+    mapping (uint256 => Bid) public bids;
 
-    VatLike public   vat;
-    bytes32 public   ilk;
+    VatLike public   vat;            // vat core accounting
```

Comment:
For consistency with other contracts, such as the `Spotter`, `Jug`, and `End`, consider changing this to `// CDP Engine`

Reply:
Changed here: https://github.com/makerdao/dss/pull/124